src/core/load-fragment.c

```diff
@@ -947,45 +951,59 @@ int config_parse_exec(
 
                 ignore = FLAGS_SET(flags, EXEC_COMMAND_IGNORE_FAILURE);
 
-                r = unit_path_printf(u, f, &path);
-                if (r < 0) {
-                        log_syntax(unit, ignore ? LOG_WARNING : LOG_ERR, filename, line, r,
-                                   "Failed to resolve unit specifiers in '%s'%s: %m",
-                                   f, ignore ? ", ignoring" : "");
-                        return ignore ? 0 : -ENOEXEC;
-                }
+                _cleanup_strv_free_ char **args = NULL;
+                _cleanup_free_ char *path = NULL;
+
+                if (FLAGS_SET(flags, EXEC_COMMAND_PREFIX_SHELL)) {
+                        /* Use /bin/sh as placeholder since we can't do NSS lookups in pid1. This would
+                         * be exported to various dbus properties and is used to determine SELinux label -
+                         * which isn't accurate, but is a best-effort thing to assume all shells have more
+                         * or less the same label. */
+                        path = strdup("/bin/sh");
```

Comment:
suggestion, maybe use libc's `_PATH_BSHELL` macro for this, given this is a magic string to some level